java/pom.xml

```diff
@@ -115,7 +115,17 @@
       <artifactId>spark-sql-kafka-0-10_${scala.binary.version}</artifactId>
       <version>${spark.version}</version>
     </dependency>
-
+    <!-- https://mvnrepository.com/artifact/com.datastax.spark/spark-cassandra-connector -->
+    <dependency>
+      <groupId>com.datastax.spark</groupId>
+      <artifactId>spark-cassandra-connector_2.12</artifactId>
+      <version>3.1.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.jnr</groupId>
+      <artifactId>jnr-posix</artifactId>
```

Comment:
are we sure that this dependency is really needed ?
What is the error without it.

Reply:
Yeah, initially I did not have it, that when I got a ClassNotFound exception within this package. The exception was within an internal class of Cassandra connector. Adding this package, solved the issue.